### ISSUE TITLE
Can't use flag  --update-labels with "="

### ISSUE THREAD

**Issue by chimanos** [google-github-actions/deploy-cloudrun#341]
### TL;DR

Hi,
I try to use the flag ` --update-labels`  in my deploy action to add some labels like this: 

`flags: --port=8080 --update-labels env=${{ inputs.ENV }}`

But when the action execute the command line, it removes the "=" to a space.

I tried to put quotes but it change nothing => `flags: --port=8080 --update-labels "env=${{ inputs.ENV }}"` but same result.

### Expected behavior

Keep the "=" characters in the flags when we use flag like ` --update-labels`.

`--port 8080 --update-labels env=prod`

### Observed behavior

Command result produce by the action => `--port 8080 --update-labels env prod`

### Action YAML

```yaml
on:
  workflow_call:
    inputs:
      docker_image:
        required: false
        type: string
      env:
        required: true
        type: string
    secrets:
      roleId:
        required: true
      secretId:
        required: true
name: Deploy to Google Cloud Run
jobs:
  deploy:
    name: Deploy to Google Cloud Run
    runs-on: ubuntu-18.04
    environment: ${{ inputs.env }}
    steps:
      # Get DOCKER_IMAGE var from other job
      - name: Import variable DOCKER_IMAGE from other jobs
        uses: UnlyEd/github-action-store-variable@v2.1.0
        if: "${{ inputs.DOCKER_IMAGE == '' }}"
        with:
          variables: |
            GAR_DOCKER_IMAGE
      # Get DOCKER_IMAGE var from inputs
      - name: Import variable DOCKER_IMAGE from inputs
        if: "${{ inputs.DOCKER_IMAGE != '' }}"
        run: echo "GAR_DOCKER_IMAGE=${{ inputs.DOCKER_IMAGE }}" >> $GITHUB_ENV
      # Import secret from Vault
      - name: Import secrets from Vault
        id: vault-secrets
        uses: hashicorp/vault-action@v2.4.0
        with:
          url: URL
          method: approle
          namespace: "namespace"
          roleId: ${{ secrets.roleId }}
          secretId: ${{ secrets.secretId }}
          secrets: |
            secret/data/ci/common GCP_CR_KEY | GCP_CR_KEY ;
      # Setup the env variable to deploy
      - name: Set env var for Docker repository
        run: |
          echo "SERVICE_NAME=${{ github.event.repository.name }}-${{ inputs.ENV }}" >> $GITHUB_ENV
      # Authent to gcp consol
      - name: Authent to GCP
        id: 'auth'
        uses: 'google-github-actions/auth@v0'
        with:
          credentials_json: ${{ steps.vault-secrets.outputs.GCP_CR_KEY }}
      # Deploy develop to GCR
      - name: 'Deploy to Cloud Run'
        uses: 'google-github-actions/deploy-cloudrun@v0'
        with:
          image: ${{ env.GAR_DOCKER_IMAGE }}
          service: ${{ env.SERVICE_NAME }}
          region: europe-west1
          flags: --port=8080 --update-labels env=${{ inputs.ENV }}
```

### Log output

```text
Running: gcloud run deploy backend-name-ref --image my-image --quiet --platform managed --region europe-west1 --port 8080 --update-labels env prod --project my-project --format json
Error: google-github-actions/deploy-cloudrun failed with: failed to execute gcloud command `gcloud run deploy backend-name-ref --image my-image --quiet --platform managed --region europe-west1 --port 8080 --update-labels env prod --project my-project --format json`: ERROR: (gcloud.run.deploy) argument --update-labels: Bad syntax for dict arg: [env]. Please see `gcloud topic flags-file` or `gcloud topic escaping` for information on providing list or dictionary flag values with special characters.
```

**Comment by sethvargo** [comment]
Hi @chimanos 

Thank you for opening an issue. `--update-labels "env=${{ inputs.ENV }}"` should have worked - can you please share the debug logs for the failed invocation?

**Comment by kenta-takeuchi** [comment]
I have the same issue and I fixed it.
Please check it out below.
https://github.com/google-github-actions/deploy-cloudrun/pull/350

**Comment by steveoh** [comment]
Is that merged pr going to create a `.11` release for this action?
